### ISSUE TITLE
Export to HTML not working

### ISSUE THREAD

**Issue by montex** [LuminanceHDR/LuminanceHDR#63]
it seems that the "Export to HTML" feature is not working properly.

I tried both the Windows and Linux versions. On Windows the program crashes ("the program has stopped working"...), and on Linux the produced HTML page is almost empty. 

Has anybody used this feature with success?

**Comment by fcomida** [comment]
It does actually work for me on both Linux and Windows (Win7).

![screenshot_20170629_142207](https://user-images.githubusercontent.com/11018726/27687326-b4688f96-5cd6-11e7-8959-440b091ae78d.png)

**Comment by fcomida** [comment]
@montex what is causing the crash on windows?  what is your linux distro? Any messages on console when running **export to html** on linux?

**Comment by montex** [comment]
@fcomida On Windows  I don't know, it just crashes while doing the export. It is Windows 10, and I downloaded the installer from the website. I am going to try on another Windows pc, maybe that is a problem of my computer, I don't use that Windows installation too much.

My linux is Debian 9 (latest). Except from this issue, Luminance HDR works fine.
This is what I get when opening the page generated in linux:

![schermata da 2017-06-30 15-36-24](https://user-images.githubusercontent.com/25503074/27737952-54f5a5ce-5daa-11e7-8e25-569aa865c588.png)

As you see I tried different browsers, but the result are the same.  
Messages on the console do not seem to be relevant:

`Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.`
`pfscopy() = 250.106 msec`
`transformColorSpace: Found right match for colorspace conversion`
`transformRGB2XYZ() = 163.099 msec`

I am attaching the files produced during the export.
[export-to-html-debian.zip](https://github.com/LuminanceHDR/LuminanceHDR/files/1115340/export-to-html-debian.zip)

**Comment by fcomida** [comment]
@montex I guess it's something related to system Locale. Try launching LHDR with: `LANG=C luminance-hdr`

**Comment by montex** [comment]
@fcomida I tried your suggestion but it did not work. I will try to investigate a bit, thanks.

**Comment by fcomida** [comment]
@montex image width and height have been written as 8.000 and 5.320 in `prova2.html` instead of 8000 and 5320. So I'm wondering a problem with system locale is somehow involved here. I'll review the code later on.

**Comment by montex** [comment]
Some updates. It is actually a locale problem as you suggested, @fcomida 
However `LANG=C luminance-hdr` was not enough, but also a `LC_ALL=C` was needed.

I created pull request #70,  it seems to solve the problem for me. Actually I realized that image dimensions were not the only problem, but also colors came out wrong (washed out), probably still because of wrong numbers resulting from locale issues.
